### PR TITLE
Ryu Flow Mod v1.0 Implemented (D111)

### PR DESCRIPTION
**Pull request by ChickenFood**

### CODE REVIEW

File: adapter/ryu/omniui/omniui.py [hunk 1]
```diff
@@ -197,15 +198,185 @@ def links(self, req, **kwargs):
             # remove bi-direction link
             reverse = False
             for link in result:
-                if(link['src-switch'] == omniLink['dst-switch'] and
-										link['dst-switch'] == omniLink['src-switch'] and
-										link['src-port'] == omniLink['dst-port'] and
-										link['dst-port'] == omniLink['src-port']):
+                if (link['src-switch'] == omniLink['dst-switch'] and
+                        link['dst-switch'] == omniLink['src-switch'] and
```
Comment (rascov) [inline adapter/ryu/omniui/omniui.py]: Too much indent spaces?

File: adapter/ryu/omniui/omniui.py [hunk 2]
```diff
@@ -197,15 +198,185 @@ def links(self, req, **kwargs):
...
+        elif cmd == 'DEL_ST':
+            cmd = dp.ofproto.OFPFC_DELETE_STRICT
+        else:
+            return Response(status=404)
+
+        if dp.ofproto.OFP_VERSION == ofproto_v1_0.OFP_VERSION:
+            flow = self.ryuFlow_v1_0(dp, omniflow)
+            ofctl_v1_0.mod_flow_entry(dp, flow, cmd)
+        elif dp.ofproto.OFP_VERSION == ofproto_v1_2.OFP_VERSION:
+            ofctl_v1_2.mod_flow_entry(dp, flow, cmd)
```
Comment (rascov) [inline adapter/ryu/omniui/omniui.py]: flow is not well defined in this if clause

File: adapter/ryu/omniui/omniui.py [hunk 3]
```diff
@@ -197,15 +198,185 @@ def links(self, req, **kwargs):
...
+
+    def to_action(self, dp, actions):
+        actions_type = actions.split('=')[0]
+        if actions_type == 'OUTPUT':
+            ryuAction = {
+                'type': actions_type,
+                'port': actions.split('=')[1],
+                'max_len': 0xffe5
+            }
+            print ("{}={}:{}".format(actions_type, actions.split('=')[1], 0xffe5))
```
Comment (rascov) [inline adapter/ryu/omniui/omniui.py]: Move unnecessary debug messages. Use unit test instead.

**Comment by ChickenFood** [comment]
1. More concise naming for line 248 (Commit 7929f1a)
2. Debug messages removed
3. STRIP_VLAN action has no value so it would yield error @ line 167 (Commit 7929f1a).
Created exception for STRIP_VLAN

**Pull request merged**